english/end/end.md

```diff
@@ -0,0 +1,8 @@
+Lol this is the dead-end page that I talked to you about ;)
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.